### PR TITLE
Proposing adding policy descriptive name

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6310

### CODE REVIEW

**Comment by nam31** [comment]
@JohanFreelancer9 please do the copy editor review for this PR.

File: Teams/sign-in-teams.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ When this policy is configured on a device, users can only sign in with accounts
 >2. The policy requires Teams for Windows version 1.3.00.30866 or higher, and Teams for MacOS version 1.3.00.30882 (released mid-November 2020).
 
 **Policies for Windows**
-Administrative Template files (ADMX/ADML) are available from the [Download center](https://www.microsoft.com/download/details.aspx?id=49030). Additionally, you can manually set keys in Windows Registry:
+Administrative Template files (ADMX/ADML) are available from the [Download center](https://www.microsoft.com/download/details.aspx?id=49030), (the policy setting descriptive name in the administrative template file is "Restrict sign in to Teams to accounts in specific tenants"). Additionally, you can manually set keys in Windows Registry:
```
Comment (JohanFreelancer9) [inline Teams/sign-in-teams.md:121]:
```suggestion
Administrative Template files (ADMX/ADML) are available from the [Download center](https://www.microsoft.com/download/details.aspx?id=49030) (the policy setting descriptive name in the administrative template file is "Restrict sign in to Teams to accounts in specific tenants"). Additionally, you can manually set keys in Windows Registry:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@MSFTTracyP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@scanum Can we request this to be merged? Thank you

**Comment by get-itips** [comment]
@nam31 Can you kindly request this to be merged? Thank you

**Comment by nam31** [comment]
@MSFTTracyP, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**